BinaryEncoding.md

```diff
@@ -117,19 +117,19 @@ The module starts with a preamble of two fields:
 | magic number | `uint32` |  Magic number `0x6d736100` (i.e., '\0asm') |
 | version | `uint32` | Version number, currently 10. The version for MVP will be reset to 1. |
 
-This preamble is followed by a sequence of sections. Each section is identified by `varint32` 
-that encodes either a known section or a user-defined section.
-Known sections have negative ids, while user-defined sections have positive ids that encode 
-the length of a string identifier immediately to follow.
-After the section identification, the section length and data follow.
+The module preamble is followed by a sequence of sections.
+Each section is identified by a 1-byte *section code* that encodes either a known section or a user-defined section.
+Known sections have non-zero ids, while user-defined sections have a zero id followed by a string identifier.
+The section length and data immediately follow the section identification.
 All sections unknown to the WebAssembly implementation are ignored.
 A validation error in a user-defined section does not cause validation for the whole module to fail and is
 instead treated as if the section was absent.
 
```

Comment:
A validation error in an 'unknown' section does not cause validation for the whole module to fail either, the 'user-defined' distinction here is not necessary.